src/test/shell/bazel/verify_workspace.sh

```diff
@@ -40,22 +40,24 @@ fi
 source "$(rlocation "io_bazel/src/test/shell/integration_test_setup.sh")" \
   || { echo "integration_test_setup.sh not found!" >&2; exit 1; }
 
-WORKSPACE_FILE="$(rlocation io_bazel/WORKSPACE)"
+WORKSPACE_FILES=("$(rlocation io_bazel/WORKSPACE)" "$(rlocation io_bazel/distdir_deps.bzl)")
 
 function test_verify_urls() {
   # Find url-shaped lines, skipping jekyll-tree (which isn't a valid URL), and
   # skipping comments.
   invalid_urls=()
-  for url in $(grep -E '"https://|http://' "${WORKSPACE_FILE}" | \
-    grep -v 'jekyll-tree' | \
-    grep -v '^ *#' | \
-    perl -pe 's#.*"(https?://[^"]+)".*#$1#g' | \
-    sort -u); do
-    #echo "Checking ${url}"
-    if ! curl --head -silent --fail --output /dev/null --retry 3 "${url}"; then
-      #fail "URL ${url} is invalid."
-      invalid_urls+=("${url}")
-    fi
+  for file in "${WORKSPACE_FILES[@]}"; do
+    for url in $(grep -E '"https://|http://' "${file}" | \
+      grep -v 'jekyll-tree' | \
+      grep -v '^ *#' | \
+      perl -pe 's#.*"(https?://[^"]+)".*#$1#g' | \
```

Comment:
Can we fix this to not use perl.  It can be done with sed.  I think
sed -e 's#.*"\(https\?://[^"]*\)".*$#\1#g'

Even better, a single command
sed -e '/jekyll-tree/d' -e '/^ *$/d' -e 's#.*"\(https\?://[^"]*\)".*$#\1#g'

Reply:
Done. I hate writing sed expressions.

Reply:
I have, on occasion, written sed *scripts* with conditional blocks and loops. It would almost always be quicker to write something in python. If only so you could have a comment block above the regex to say what you are doing.